Traidoo/settings/django/middleware.py

```diff
@@ -8,7 +8,6 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
-    "reversion.middleware.RevisionMiddleware",
```

Comment:
I would delete it completely 🤔

Reply:
You mean:

- [ ] remove from requirements
- [ ] remove tables from database in migrations

Reply:
Exactly. 😉

Reply:
Done.

Reply:
Removed also from admin.